### DISCUSSION TITLE
Dual Display video different for each register

### DISCUSSION THREAD

**Discussion by SuryaMSSupp** [microsoft/Dynamics365Commerce.Solutions#170] [Q&A] [answered]
Good day!
Can you please help me in answering the below question:

Issue:
We had a requirement to display the video in dual display form based on the register, so the customer would like to see that they would see different video on different registers, customer would like to know how to achieve this, they could successfully test seeing a video on a dual display per Store by having the setup on the hardware profile, but now we would like to check if they can configure different dual display videos for different registers.

**Comment by madyke** [comment, marked as answer]
Hello @SuryaMSSupp - there are a couple of options the customer can use here:

- You mentioned configuring the video per store using the hardware profile, but you can also set the hardware profile on an individual register. So they could create different hardware profiles, one for each video they want to display, and then for each register configure it with the hardware profile that has the video they want that display to use. 
- They could also implement a CRT trigger for `GetHardwareProfile` that runs after the request. The context for that trigger would have device ID info, so they could then call into a mapping in code that pairs each device ID to the video they want to display. Then they just need to update the hardware profile response in the trigger with the video they want to display for that particular register.
[https://learn.microsoft.com/en-us/dynamics365/commerce/dev-itpro/commerce-runtime-extensibility-trigger](https://learn.microsoft.com/en-us/dynamics365/commerce/dev-itpro/commerce-runtime-extensibility-trigger)

**Reply by pruthvigowda483** [reply]
Hi Madyke,

Thanks for your reply and suggestions.
I have a few doubts on second point, let me first explain you a requirement, we have requirement that when we login to the pos it needs to display a video in the dual display screen based on the store or register level which means there are 10 register means 10 video links will be there that we have created a custom field in the register form, from there we need to pick the video links to display for each register, i don't think the above suggestion will work, if it will work can you please give more details.

**Reply by madyke** [reply]
Hi @pruthvigowda483 - the two options I gave above are distinct, so you only need to pick one - if the second doesn't sound like it will work the first point is still an option.

I'm not sure I understand the requirement you're trying to explain or how it differs from how it was explained in the original post (basically each register needs to have a different video running on its associated dual display). If you have a link to each video that you want to display for different registers, that link can be defined in the Hardware Profile for that specific register under the Dual Display section. That would require setting up different Hardware Profiles for each register, which may be a lot of overhead to create and manage going forward, but it would solve the problem if you don't want to do it in code like the second bullet point explains.
